From 31f13d30b35577ba37390ad05227b59b3254cd78 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:08:20 -0400
Subject: [powerpc] Kconfig: Make TM select VSX and VMX

Message-id: <1410545655-205645-272-git-send-email-dzickus@redhat.com>
Patchwork-id: 94217
O-Subject: [RHEL7 PATCH 271/626] powerpc/Kconfig: Make TM select VSX and VMX
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit 7b37a1232273912dd57cd72b82fe70407bff7683
Author: Michael Neuling <mikey@neuling.org>
Date:   Wed Jan 8 21:25:31 2014 +1100

    powerpc/Kconfig: Make TM select VSX and VMX

    There are no processors in existence that have TM but no VMX or VSX.  So let's
    makes CONFIG_PPC_TRANSACTIONAL_MEM select both CONFIG_VSX and CONFIG_ALTIVEC.
    This makes the code a lot simpler by removing the need for a bunch of #ifdefs.

    Signed-off-by: Michael Neuling <mikey@neuling.org>
    Signed-off-by: Paul Mackerras <paulus@samba.org>
    Signed-off-by: Alexander Graf <agraf@suse.de>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/Kconfig b/arch/powerpc/Kconfig
index fcc8604..a446336 100644
--- a/arch/powerpc/Kconfig
+++ b/arch/powerpc/Kconfig
@@ -324,6 +324,8 @@ config PPC_TRANSACTIONAL_MEM
        bool "Transactional Memory support for POWERPC"
        depends on PPC_BOOK3S_64
        depends on SMP
+       select ALTIVEC
+       select VSX
        default n
        ---help---
          Support user-mode Transactional Memory on POWERPC.
-- 
1.7.1